packages/node/src/node.ts

```diff
@@ -55,6 +56,21 @@ export class Node {
       `${nodeConfig.STORE_KEY_PREFIX}/${this.signer.address}`
     );
     this.instructionExecutor = new InstructionExecutor(networkContext);
+    this.instructionExecutor.middlewares.add(
+      Opcode.STATE_TRANSITION_COMMIT,
+      async (message, next, context) => {
```

Comment:
I think you need to add the types here? Does type inference work?

Reply:
It does! Though I prefer adding types anyway. My IDE autogen'd this

Reply:
I would really prefer passing them through return types rather than binding to `STATE_TRANSITION_COMMIT`. If this can be done the opcode should be deleted.

Reply:
What does "passing them through return types" mean?

Reply:
This is a separate issue that has been discussed in the past; I'd recommend we open an issue to discuss it, probably to be done after the Playground release.